technicalreport/stateoftheart.tex

```diff
@@ -437,104 +435,83 @@ \subsubsection{Berechnungsmethodiken ÖV-Güteklassen}
 Dabei behält man das Ziel der automatischen Berechnung hinter dem geistigen Auge.
 Ob die definierten Kriterien mit der vorhandenen Datenbasis umsetzbar sind und ob diese für eine schweizweite Umsetzung tauglich ist, ist Bestandteil dieser Analyse.
-Bei den Optimierungen lehnen wir uns ebenfalls stark an die Erkenntnisse von~\cite{oev-guteklasse-gr}, welche die Norm an die spezifische Bedürfnisse des Kantons Graubünden anpasst hat und ~\cite{berechnung_are}, welche die Berechnungsmethodik basierend auf dieser Norm mit eigenen Anpassungen umgesetzt hat.
+Basierend auf diesen Verbesserungsmöglichkeiten, der \nameref{Lösungsansätze:Berechnungsmethodik ARE} und den Learnings der beiden kantonalen Berechnungsmethodiken wird im Kapitel \ref{Spezifikation OeVGK18} eine neue Berechnungsmethodik vorgeschlagen.
+
+\subsubsection{Ermittlung der Erreichbarkeit der Haltestelle}
+\label{Verbesserungsmöglichkeiten:Ermittlung der Erreichbarkeit der Haltestelle}
+
+\paragraph{Problem}~\\
+In allen Lösung wird die Luftlinie als massgebende Bewertung der Erreichbarkeit einer Haltestelle genommen.
+Dies berücksichtigt weder die zusätzlicher Distanz durch verwinkelte Wege noch die Topographie.
+Der Kanton Graubünden behebt diese Problematik mit manuellen Anpassungen.
+Manuelle Eingriffe eigenen sich für eine automatisierte Lösung nicht.
+
+\paragraph{Lösung}~\\
+Um der Topographie und der Wegführung gerecht zu werden, wird einerseits der konkrete Weg mit all seinen Eigenschaften für die Distanzberechnung berücksichtigt.
+Auf der anderen Seiten wird die Topographie durch die Berechnung von \gls{Leistungskilometer}n berücksichtigt.
+
+\subsubsection{Kursintervallberechnung}
+\label{Verbesserungsmöglichkeiten:Kursintervallberechnung}
+% TODO gefällt mir noch nicht so, zu wenig detailliert
+
+\paragraph{Problem}~\\
+Die Methodiken verwenden grundsätzlich einen Werktag ausserhalb der Ferienzeit und der touristischen Hochsaison und einen Bereich, welcher den Pendlerverkehr zu den Randzeit berücksichtigt, um den Kursintervall berechnen zu können.
+Der Kanton Zürich geht noch einen Schritt weiter und vergrössert ihren Einzugsbereich.
+Die Kurse am Wochenende und in der Nacht werden grundsätzlich ausgeklammert.
+Für Privatpersonen, aber auch für Planer kann dieser Zeitraum relevant sein, um sich für einen Wohnort entscheiden oder um ein Angebot ausbauen zu können.
 
-\subparagraph{Transportmittelkategorie}~\\
-Die öffentlichen Verkehrsmittel werden in vier Kategorien gruppiert (siehe Tabelle \ref{table:Transportmittelkategorien}).
+\paragraph{Lösung}~\\
+Die Berechnung der \acs{ÖV}-Güteklassen wird für drei Zeiträume durchgeführt.
+Zusätzlich zum Standardbereich, welcher den Pendlerverkehr berücksichtigt, wird der Verkehr ausserhalb des Pendlerverkehrs in der Nacht und das Wochenende berücksichtigt.
+Natürlich muss hier eine andere Gewichtung angewendet werden, da zu Randzeiten Stundentakte weit häufiger vorkommen.
```

Comment:
Heisst das, wir definieren für Randzeiten andere Kategorisierungen der Güteklassen? Fände ich etwas schwierig, da wir uns auf nichts stützen können. Dies würde auch den Vergleich schwieriger machen, für mich wäre es OK, wenn die Güteklassen zu Randzeiten einfach schlechter sind, nur so kann man direkt z.B. zwischen Tag und Nacht vergleichen

Reply:
Lassen wir in einer ersten Spezifikation aussen vor und besprechen das im ersten Review